### ISSUE TITLE
Add numberOfDays to ContainerHeader

### ISSUE THREAD

**Issue by petemusicpancake** [howljs/react-native-calendar-kit#138]
Hi!

It would be to have an option to change numberOfDays for CalendarContainer and CalendarHeader separately. I need to show more days in header, but only single day view for body

**Comment by howljs** [comment]
I’m not quite sure what your feature will look like; it seems a bit unique. You can customize it by using the `renderHeaderItem` in CalendarHeader

**Comment by petemusicpancake** [comment]
> I’m not quite sure what your feature will look like, it seems a bit unique. You can customize it by using the `renderHeaderItem` in CalendarHeader

I wanted to achieve CalendarHeader to have 7 days, while showing 1 day view in Body.

**Comment by howljs** [comment]
If that’s the case, you might have to build a separate CalendarHeader. You can use a custom hook like useCalendar, similar to how I’m doing it. It will take a bit more time, but it will be the way you want

<img width="964" alt="Screenshot 2024-10-18 at 21 58 12" src="https://github.com/user-attachments/assets/f02019db-89b3-4f5b-9622-fffd7b88444f">

In the near future, I will consider building an additional Header component similar to Apple Calendar

**Comment by petemusicpancake** [comment]
Yeah, I just created my custom componen. It looks fine, except it doesn't have that paging animation when week changes. 
>In the near future, I will consider building an additional Header component similar to Apple Calendar
That would be great. 

Anyways, thank you for the quick reply and a great library

**Comment by CarnageXD** [comment]
@petemusicpancake Hey! This is exactly what I'm also looking for! Could you please share a code snippet of your header and how you sync this with the CalendarBody?

**Comment by vhavryliv** [comment]
Hi, @howljs, @CarnageXD !
faced the problem of displaying events for the whole day with the same kind of calendar. Maybe you had the same?
https://github.com/howljs/react-native-calendar-kit/issues/200
